pkg/segment/utils/number_test.go

```diff
@@ -0,0 +1,402 @@
+package utils
+
+import (
+	"math"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// Test_isInvalid tests if the number is invalid. It should return invalidType.
+func Test_isInvalid(t *testing.T) {
+	n := Number{}
+	n.SetInvalidType()
+	assert.True(t, n.ntype() == invalidType)
```

Comment:
This you can change to:
`assert.Equal(t, invalidType, n.ntype())`

Reply:
Same comment elsewhere

Reply:
changed assert.True to Equal everywhere where applicable